tests/loggen/loggen.c

```diff
@@ -463,8 +469,18 @@ main(int argc, char *argv[])
   signal(SIGPIPE, signal_callback_handler);
   setup_rate_change_signals();
 
-  int plugin_num = enumerate_plugins(SYSLOG_NG_PATH_LOGGEN_PLUGIN_DIR, plugin_array, ctx);
-  DEBUG("%d plugin successfuly loaded\n", plugin_num);
+  /* if plugin path defined by environment variable then we should use it */
+  const char *plugin_path = g_getenv(ENV_SYSLOGNG_LOGGEN_PLUGINDIR);
```

Comment:
I think the `SYSLOGNG_PREFIX` serves the same purpose. Using only that should be sufficient, even if that requires the additional `/lib/syslog-ng/loggen` suffix. (keep this simple)